webpack/assets/stylesheets/application/partials/_base.scss

```diff
@@ -105,12 +105,12 @@ a.config_group_name {
 }
 
 .editable {
-  background: image-url('edit.png') no-repeat scroll 98% 6px transparent;
+  background: url('#{$local-img-path}/edit.png') no-repeat scroll 98% 6px transparent;
```

Comment:
we can probably remove this file and change it to an icon (looks like a leftover)

Reply:
Won't be that trivial, check out:
https://github.com/theforeman/foreman/blob/develop/app/views/settings/index.html.erb#L30
https://github.com/theforeman/foreman/blob/develop/app/helpers/settings_helper.rb#L2-L11